services/harmony/test/service-image-tags.ts

```diff
@@ -652,6 +700,86 @@ describe('Service image endpoint', async function () {
         deploymentId = getDeploymentIdFromStatusLink(link);
         expect(deploymentId).to.not.be.null;
       });
+
+      it('reaches a terminal status without timeout', async function () {
+        const noTimeout = await waitUntilStatusChange(deploymentId);
+        expect(noTimeout).to.be.true;
+      });
+
+      describe('when get the service image tag update state after a successful service deployment', async function () {
+        before(async function () {
+          hookRedirect('adam');
+          this.res = await request(this.frontend).get('/service-image-tag/state').use(auth({ username: 'adam' }));
+        });
+
+        after(function () {
+          delete this.res;
+        });
+
+        it('returns a status 200', async function () {
+          expect(this.res.status).to.equal(200);
+        });
+
+        it('returns the service image enabled true', async function () {
+          expect(this.res.body).to.eql({
+            'enabled': true,
+          });
+        });
+      });
+    });
+  });
+
+  describe('Get service deployment enabled state permission test', async function () {
```

Comment:
In case you were wondering, I tried to consolidate the permission test on all GET endpoints into one, e.g. 
```
describe('Get endpoints permission test', async function () {
    const routes = [{ path: '/service-image-tag', expectedResult: serviceImages },
    { path: '/service-image-tag/state', expectedResult: { enabled: true } },
    { path: '/service-image-tag/deployment/xxxx', expectedResult: 'yyy' }];

    routes.forEach(route => {
       ......
```
But the test is not as readable because the different status code and fields that need to be verified. So I ended up with separate tests for better readability.